flexible/java/samples/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,31 @@
+runtime: java
+env: flex
+
+handlers:
+- url: /.*
+  script: this field is required, but ignored
+
+use_memcache_proxy: true
+
+# [START env_variables]
+env_variables:
```

Comment:
What I like to do is provide credentials at build time `mvn package -DSENDGRID_API_KEY=bha`